doc/tutorial/debug_faq.txt

```diff
@@ -214,6 +214,13 @@ The ``compute_test_value`` mechanism works as follows:
 * ``ignore``: Silently ignore the computation of intermediate test values, if a
   variable is missing a test value.
 
+Theano also provides the ``print_test_value`` print flag which can hold a boolean value.The default value is false.If 'True', Theano will override the ‘__str__’ method of its variables to also print the tag.test_value when this is available.
```

Comment:
I think this is a little better:

Theano also provides the ``print_test_value`` flag which can take a boolean value.The default value is ``False``. If ``True``, Theano will print the test value when printing Theano variable and it have a test value.

I'm not a native english, maybe is can be made even better. Don't hesitate to change my proposition.